subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -163,6 +173,12 @@ static bool validate_firmware(u32_t fw_dst_address, u32_t fw_src_address,
 					(u8_t *)fw_val_info->firmware_address,
 					fwinfo->firmware_size);
 
+		if ((retval == 0)/* && !external*/) {
```

Comment:
I assume this should be fixed now as this is not DNM or draft anymore?

Reply:
This is waiting for https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1223/files#diff-697e1610d63057fdcccec505062c7bcdR94 which adds the `external` parameter.

Reply:
Removed the commented out code.